src/content/development/contribution-guide/_index.en.md

```diff
@@ -56,9 +56,41 @@ To be expanded
 
 To be expanded
 
-#### Pre-Planning
+#### Planning
 
-To be expanded
+The week before a new release cycle starts is dedicated to planning the release.
+Planning accounts for [epics], [tasks] and [bugs] which we intend to handle in the next version.
+Planning meetings will be held, focusing on the [Backlog board].
+
+##### Inclusion Criteria
+
+For a [task] or [epic] to be eligible for the next version, it needs to fulfill these requirements:
+
+* Be part of the [Backlog board].
+* Have a `next-version-candidate` label.
+* Have a description detailing **what** the issue is and optionally **how** it's going to be solved.
+* Have an appropriate sizing label, according to the amount of work expected for a single person to completely deliver the task:
+  * *Small*: Work is contained in one sprint and is expected to take less than half the sprint.
+  * *Medium*: Work is contained in one sprint and is expected to take most of the sprint.
+  * *Large*: Work is contained within a release (two-three sprints).
+  * *Extra-Large* = Work can't be contained within a release and would span multiple releases.
+* Any *Large* or *Extra-Large* task must be converted to an [epic].
+* In case of an [epic]:
+  * Be an issue in the [enhancements] project.
+  * Adhere to the [epic template].
+  * Have a high level break down of the **expected** work, corresponding to the "Definition of Done".
+
+##### Planning Meetings
+
+The project team will hold planning meetings, led by the project's "scrum master".
+During these meetings, the project team will:
```

Comment:
The term "master" isn't considered politically correct these days. How about we just avoid mentioning who leads the meeting.

```suggestion
The project team will hold planning meetings to:
```

Reply:
As I replied to Nir already, this is the adjective form and is totally acceptable - https://github.com/submariner-io/submariner-website/pull/720/files#r832008284

Reply:
This isn’t an adjective.

Reply:
No it isn't.